doc/lua_api.txt

```diff
@@ -4938,18 +4938,22 @@ This is basically a reference to a C++ `ServerActiveObject`
     * element `stat` values:
       `position`, `name`, `scale`, `text`, `number`, `item`, `dir`
 * `hud_get(id)`: gets the HUD element definition structure of the specified ID
-* `hud_set_flags(flags)`: sets specified HUD flags to `true`/`false`
-    * `flags`: (is visible) `hotbar`, `healthbar`, `crosshair`, `wielditem`,
-      `breathbar`, `minimap`, `minimap_radar`
-    * pass a table containing a `true`/`false` value of each flag to be set or
-      unset.
-    * if a flag equals `nil`, the flag is not modified
-    * note that setting `minimap` modifies the client's permission to view the
-      minimap - the client may locally elect to not view the minimap.
-    * minimap `radar` is only usable when `minimap` is true
-* `hud_get_flags()`: returns a table containing status of hud flags
-    * returns `{hotbar=true, healthbar=true, crosshair=true, wielditem=true,
-      breathbar=true, minimap=true, minimap_radar=true}`
+* `hud_set_flags(flags)`: sets specified HUD flags of player.
+    * `flags`: A table of player HUD flags with boolean values.
+    * List of flags that can be toggled:
+        * hotbar
+        * healthbar
+        * crosshair
+        * wielditem
+        * breathbar
+        * minimap
+        * minimap_radar
+    * If a flag equals `nil`, the flag is not modified
+    * `minimap`: Modifies the client's permission to view the minimap.
+      The client may locally elect to not view the minimap.
+    * `minimap_radar` is only usable when `minimap` is true
+* `hud_get_flags()`: returns a table of player HUD flags with boolean values.
```

Comment:
It's clearest to explain the form is identical to that used in 'set flags'. Like:
`'get_physics_override()': returns the table given to 'set_physics_override'`

Reply:
`get_physics_override` returns exactly the same table that was previously passed to `set_physics_override`. In the case of `hud_get_flags` however, the table returned is _not _ the same as the table passed to it's corresponding setter, but is one with _all_ the HUD flags and their respective boolean values. Also, the table passed to `hud_set_flags` allows fields with `nil` values - i.e. omitted fields.

Reply:
Good point.

Reply:
Ok this line is ok for me.

Reply:
Done